src/app/ubs/ubs-admin/services/order.service.ts

```diff
@@ -92,7 +92,7 @@ export class OrderService {
     }
   }
 
-  getOrderInfo(orderId) {
+  getOrderInfo(orderId: number) {
```

Comment:
To improve type safety and avoid using any, you can use an interface `IOrderInfo` 
you can adjust the return type to `Observable<IOrderInfo>`

Pay attention on some methods like: 
`updateRecipientsData, getOrderDetailStatus, getAllResponsiblePersons, getAllReceivingStations, getOrderExportDetails, readAddressOrder, getPaymentInfo, getUserViolations, getOrderSumDetails, getOrderDetails ` are candidates for refactoring or removal